ecssd_agent.go

```diff
@@ -249,14 +263,8 @@ func deleteDNSRecord(serviceName string, dockerId string) error {
 					ResourceRecordSet: &route53.ResourceRecordSet{
```

Comment:
This seems to send a command to delete everything EXCEPT the current IP?

I thought that this would be a change command to delete ONLY the current IP.